build_taglib.bat

```diff
@@ -32,50 +32,25 @@ mkdir %PLATFORM%
 cd %PLATFORM%
 REM NOTE(pegasus): generated solution with
 
-"%CMAKEDIR%\cmake" ../ -G %COMPILERPROJECT% -DBUILD_SHARED_LIBS=%DYNAMIC% -DCMAKE_SUPPRESS_REGENERATION=1 -DCMAKE_USE_RELATIVE_PATHS=1 -DZLIB_INCLUDE_DIR=%INCLUDE_DIR% -DZLIB_LIBRARY=%LIB_DIR%\%ZLIBNAME%
+"%CMAKEDIR%\cmake" .. -G %COMPILERPROJECT% -DBUILD_SHARED_LIBS=%DYNAMIC% -DCMAKE_SUPPRESS_REGENERATION=1 -DZLIB_INCLUDE_DIR=%INCLUDE_DIR% -DZLIB_LIBRARY=%LIB_DIR%\%ZLIBNAME% -DCMAKE_PREFIX_PATH=%ROOT_DIR%\Qt-%QT_VERSION%\lib\cmake -DCMAKE_INSTALL_PREFIX=%ROOT_DIR% -DCMAKE_INSTALL_INCLUDEDIR=%INCLUDE_DIR%
 IF ERRORLEVEL 1 (
     SET VALRETURN=1
 	goto END
 )
  
-%MSBUILD% taglib.sln /p:Configuration=%CONFIG% /p:Platform=%PLATFORM% /t:tag:Clean;tag:Rebuild
+"%CMAKEDIR%\cmake" --build . --config %CONFIG% --clean-first 
 IF ERRORLEVEL 1 (
     SET VALRETURN=1
 	goto END
 )
 
-copy taglib\%CONFIG%\tag.lib %LIB_DIR%
-if NOT %STATIC_LIBS% ( copy taglib\%CONFIG%\tag.dll %LIB_DIR% )
-copy taglib\%CONFIG%\tag.pdb %LIB_DIR%
-cd ..
-md %INCLUDE_DIR%\taglib
-%XCOPY% taglib_config.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ape\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\asf\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\flac\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\it\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mod\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mp4\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mpc\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mpeg\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mpeg\id3v1\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mpeg\id3v2\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\mpeg\id3v2\frames\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ogg\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ogg\flac\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ogg\opus\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ogg\speex\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\ogg\vorbis\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\riff\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\riff\aiff\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\riff\wav\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\s3m\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\toolkit\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\toolkit\*.tcc %INCLUDE_DIR%\taglib
-%XCOPY% taglib\trueaudio\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\wavpack\*.h %INCLUDE_DIR%\taglib
-%XCOPY% taglib\xm\*.h %INCLUDE_DIR%\taglib
+"%CMAKEDIR%\cmake" --build . --target install 
```

Comment:
Does this need a `--config %CONFIG%`? Looking at the build logs, this seems to trigger build in CONFIG=Debug mode as well.

Reply:
Looks like it's installing the Debug version:
```
  -- Install configuration: "Debug"
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/bin/taglib-config.cmd
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/lib/pkgconfig/taglib.pc
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/lib/tag.lib
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tag.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/fileref.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/audioproperties.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/taglib_export.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/taglib_config.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/taglib.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tstring.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tlist.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tlist.tcc
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tstringlist.h
  -- Installing: C:/jenkins/bs-2.2-win-prs/amd64-ReleaseFastbuild/include/taglib/tbytevector.h
...
```

Reply:
Ah, right. I will fix that.